compat/mimalloc/alloc-aligned.c

```diff
@@ -0,0 +1,260 @@
+/* ----------------------------------------------------------------------------
+Copyright (c) 2018-2021, Microsoft Research, Daan Leijen
+This is free software; you can redistribute it and/or modify it under the
+terms of the MIT license. A copy of the license can be found in the file
+"LICENSE" at the root of this distribution.
```

Comment:
We don't appear to have a copy of the MIT license included in this directory. Should we? What's the rule here for labeling the code as MIT within the GPL tree?

Reply:
I guess we could. We did that with nedmalloc, but we did not do that with `poll`. Probably because of the "more compatible" license of the latter?